codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/BuilderGeneratorBase.java

```diff
@@ -280,17 +280,17 @@ protected void renderBuildersForOperation(OperationShape operation, Shape inputS
 
         writer
                 .write("")
-                .write("# Operation Builder for $L", operation.getId().getName())
                 .openBlock("class $L", symbol.getName())
                 .call(() -> renderOperationBuildMethod(operation, inputShape))
                 .closeBlock("end");
 
-        LOGGER.finer("Generated operation builder for: " + operation.getId().getName());
+        LOGGER.finer("Generated builder for operation " + operation.getId().getName());
     }
 
     /**
      * @param inputShape inputShape from a streaming operation to render for.
      */
+    // TODO: should probably be in Rest Base
```

Comment:
Yeah, this is where this breaks down a little bit.  We do rely on this being in all of our existing, HTTP protocols.  We could add another level:  Rest -> Http -> Base -> Interface.

Reply:
Yeah. I think this is a bigger refactor and it's not just Builder generator but also Errors, Stubs, Parsers, etc. For now I just put a comment but we should revisit especially when doing mqtt.